apparmor.d/groups/kde/kcminit

```diff
@@ -12,6 +12,8 @@ profile kcminit @{exec_path} {
   include <abstractions/gtk>
   include <abstractions/kde-strict>
 
+  unix (connect, receive, send) type=stream peer=(addr="@/tmp/.X11-unix/X[0-9]*"),
```

Comment:
That should already be included in `X-strict` (that in included by `kde-strict`)